### ISSUE TITLE
Recalls: Keyboard focus indicator is not visible for the "Brand" content

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#156]
URL: https://rsams.dev.cloud.openplus.ca/en/alert-recall/transport-canada-recall-2022036-jayco and
User Agent: Firefox (v99), Chrome (v100), MS Edge
Language: English and French

Issue Description: When tabbing through, the keyboard focus indicator is not visible for the "Brand(s)" collapsed/expand content.

![image](https://user-images.githubusercontent.com/12256067/162053010-d9101a68-ed6f-45fe-81a2-7b88c1c1de8b.png)

Steps to reproduce:

1. Open the URLs provided above
2. Navigate the page using the "Tab" key

Expected results: The keyboard focus indicator should be visible for that element.

**Comment by sarasuri** [comment]
Verified (Sept 27, 2022) on DEV and PROD, the issue is fixed on both sites.
DEV: https://rsams.dev.cloud.openplus.ca/en/alert-recall/transport-canada-recall-2022036-jayco
PROD: https://recalls-rappels.canada.ca/en/alert-recall/novum-iq-large-volume-pump

Closing the issue.
